scripts/src/workflowtesting/checkprforci.py

```diff
@@ -0,0 +1,84 @@
+import re
+import argparse
+import os
+import requests
+import json
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+def check_if_ci_only_is_modified(api_url):
+    # api_url https://api.github.com/repos/<organization-name>/<repository-name>/pulls/1
+
+    files_api_url = f'{api_url}/files'
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    pattern_workflow = re.compile(r".github/workflows/.*")
+    pattern_script = re.compile(r"scripts/.*")
+    pattern_test = re.compile(r"tests/.*")
+    page_number = 1
+    max_page_size,page_size = 100,100
+
+    workflow_found = False
+    other_found = False
+    test_config_path = ""
+
+    while (page_size == max_page_size) & (not other_found):
```

Comment:
Should this be `and` instead of the bitwise `&`?

Reply:
second condition removed - no longer needed - see below.